pkg/generator/generator.go

```diff
@@ -64,6 +66,12 @@ func (g *Generator) SelectStmt(options Options) (string, error) {
 		selectStmt += fmt.Sprintf(" WHERE (%s)", strings.Join(exprGroup, ") AND ("))
 	}
 
+	// control the max count of order by clause
+	orderBy := g.RdOrderBy(columnsList, 2)
```

Comment:
I limit only 2 `order by` fields now, but it can be widened if need in the future.